src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DeleteOperationTest.java

```diff
@@ -219,7 +225,10 @@ public void deleteWithDynamic() {
               1,
               1,
               ReadType.KEY,
-              objectMapper);
+              objectMapper,
+              null,
+              0,
+              0);
```

Comment:
we could have created a shortcut constructors.. or static method providers..

Reply:
Will refactor code as separate PR.